cfgov/cfgov/settings/production.py

```diff
@@ -87,23 +87,7 @@
 CACHES.update({
     'default': {
         'BACKEND': 'django.core.cache.backends.filebased.FileBasedCache',
-        'LOCATION': '/tmp/eregs_cache',
-    },
-    'eregs_longterm_cache': {
-        'BACKEND': 'django.core.cache.backends.filebased.FileBasedCache',
-        'LOCATION': '/tmp/eregs_longterm_cache',
-        'TIMEOUT': 60*60*24*15,     # 15 days
-        'OPTIONS': {
-            'MAX_ENTRIES': 10000,
-        },
-    },
-    'api_cache': {
-        'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
-        'LOCATION': 'api_cache_memory',
-        'TIMEOUT': 3600,
-        'OPTIONS': {
-            'MAX_ENTRIES': 1000,
-        },
+        'LOCATION': '/tmp/django_cache',
```

Comment:
Although this is a great change to make, I fear that this might cause problems during deployment. We could discuss/address this in another forum, but essentially, while the current servers happen to have `/tmp/eregs_cache` and `/tmp/eregs_longterm_cache` set up, I don't believe these are covered by the deployment scripts (e.g. Ansible).

See also internal platform#578. We could manually create a `/tmp/django_cache` or perhaps add that step to the Ansible deploy, but I might actually proposing removing the default file-based cache in favor of a database-based cache, since we're already using that for the post preview and mega menu (tables `post_preview_cache` and `default_fragment_cache`). We could also consolidate all of these uses into a single database cache table (which would be the new `default` cache), distinguishing their content by cache key prefix.

Reply:
I'll leave it alone for now.

![saddest_face_ever_by_loveless_nights-d66ra4b](https://user-images.githubusercontent.com/10562538/45495074-84e21880-b740-11e8-9375-6ce3422fea06.png)

> See also internal platform#578. We could manually create a /tmp/django_cache or perhaps add that step to the Ansible deploy, but I might actually proposing removing the default file-based cache in favor of a database-based cache, since we're already using that for the post preview and mega menu (tables post_preview_cache and default_fragment_cache). We could also consolidate all of these uses into a single database cache table (which would be the new default cache), distinguishing their content by cache key prefix.

This sounds like an excellent idea.